docs/docs-content/clusters/cluster-management/cluster-proxy.md

```diff
@@ -0,0 +1,223 @@
+---
+sidebar_label: "Configure Applications to Use Proxy Server"
+title: "Configure Applications to Use Proxy Server"
+description: "Guide to configure configurations in a cluster to use a proxy server."
+hide_table_of_contents: false
+sidebar_position: 201
+tags: ["clusters", "cluster management", "proxy"]
+---
+
+You can configure applications that operate in your Palette clusters to use a proxy server to access the internet
+instead of accessing the internet directly.
+
+Before you are able to configure applications in a cluster to use the proxy server, you must first ensure that the
+cluster itself is configured to use the proxy server. This means that the host Operating System (OS) and the Palette
+agent inside the cluster will use the proxy server for its outbound communications. The steps to do this vary depending
+on your environment.
+
+After the cluster is configured to use the proxy server, you can proceed to configure the applications inside the
+cluster to use the proxy server. You can do this by applying the `spectrocloud.com/connection: proxy` label to the
+specific job, deployment, or daemon set to instruct an application to use the proxy settings of the cluster.
+
+## Prerequisites
+
+- An active proxy server reachable by your cluster.
+
+<Tabs groupId="scenario">
+
+<TabItem value="Palette SaaS Non-Edge">
+
+- A PCG is deployed into an active and healthy Kubernetes cluster. Refer to
+  [Deploy a PCG to an Existing Kubernetes Cluster](../pcg/deploy-pcg-k8s.md) for additional guidance.
+
+  :::warning
+
+  If you deployed a [PCG through the Palette CLI](../pcg/pcg.md#supported-environments), refer to the respective
+  platform installation guide for instructions on how to configure proxy settings during the installation process
+  through the CLI.
+
+  :::
+
+- The PCG is configured to use the proxy server that you intend for your applications to use for outbound
+  communications. For more information, refer to
+  [Enabled and Manage Proxy Configurations](../pcg/manage-pcg/configure-proxy.md).
```

Comment:
Maybe we can combine these two into a single prereq and avoid the warning. It's a step that must be done, but how it is done depends on the PCG install method.